Problems/10773/ans_10773_BNR.PY

```diff
@@ -0,0 +1,25 @@
+'''
+https://www.acmicpc.net/problem/10773
+
+스택
+LIFO(Last In First Out)
+
+'''
+import sys
+
+def solve():
+    K = int(sys.stdin.readline())
+    input_list=[]
+    stack_list=[]
+    for _ in range(K):
+        input_list.append(int(sys.stdin.readline()))
+        
+    for i in input_list:
+        if i != 0:
+            stack_list.append(i)
+        else:
+            del stack_list[-1]
```

Comment:
```suggestion
            stack_list.pop()
```
pop을 받아주는게 없긴 하다만, 좀 더 stack의 포인트에 맞는 구문이라 생각.

Reply:
저도 저기 pop() 추가하는거 말고는 더 리뷰 할 건 없어보이네요